lib/reporter.js

```diff
@@ -0,0 +1,45 @@
+var chalk = require('chalk');
+var _ = require('lodash');
```

Comment:
Consider just bundling `lodash.difference` rather than the full lot. :smile:

https://www.npmjs.com/package/lodash.difference